text.go

```diff
@@ -0,0 +1,236 @@
+/* text.go contains functions for text annotation
+TODO:
+getters and setters for
+- Stretch (redefine C type)
+- Style (redefine C type)
+- Resolution (two doubles)
+- Decoration (redefine C type)
+- Encoding (string)
+-  DrawSetTextInterlineSpacing(DrawingWand *,const double),
+-  DrawSetTextInterwordSpacing(DrawingWand *,const double),
+-  DrawSetGravity(DrawingWand *wand,const GravityType gravity)
+*/
+
+package canvas
+
+/*
+#cgo CFLAGS: -fopenmp -I./_include
+#cgo LDFLAGS: -lMagickWand -lMagickCore
+
+#include <wand/magick_wand.h>
+*/
+import "C"
+
+import (
+	"unsafe"
+)
+
+type Alignment uint
+
+const (
+ 	UndefinedAlign Alignment	= Alignment(C.UndefinedAlign)
+	LeftAlign					= Alignment(C.LeftAlign)
+	CenterAlign					= Alignment(C.CenterAlign)
+	RightAlign					= Alignment(C.RightAlign)
+)
+
+// structure containing all text properties for an annotation
+// except the colors that are defined by FillColor and StrokeColor
+type TextProperties struct {
+	Font		string
+	Family		string
+	Size		float64
+	// Stretch		C.StretchType
+	Weight		uint
+	// Style		C.StyleType
+	// Resolution  [2]C.double
+	Alignment	Alignment
+	Antialias	bool
+	// Decoration	C.DecorationType
+	// Encoding	string
+	Kerning		float64
+	// Interline	float64
+	// Interword	float64
+	UnderColor	*C.PixelWand
+}
+
+// Returns a TextProperties structure.
+// Parameters:
+//   read_default: if false, returns an empty structure.
+//				   if true, returns a structure set with current canvas settings
+func (self *Canvas) NewTextProperties(read_default bool) *TextProperties {
+	if read_default == true {
```

Comment:
I think it's more Go style to have `readDefault` instead of `read_default`. Also, you don't need the `== true` in this case.